blueprints/model/files/__root__/__path__/__name__.js

```diff
@@ -1,5 +1,5 @@
 <%= importStatements %>
 
 export default Model.extend({
-  <%= attrs %>
+<%= attrs.length ? '  ' + attrs : '' %>
```

Comment:
@pangratz Thanks for the feedback!

The problem I run into with the `if` condition is that if I do pass in attrs 
`ember g model foo --attrs name, title`
I get empty lines surrounding them:

```
export default Model.extend({

  name,: attr(),
  title: attr()

});
```

The ternary doesn't have that issue.  I tried the suggestion found here: https://github.com/emberjs/data/issues/4306#issuecomment-218454549 using `-%>` but received an `Unexpected token` error no matter where I used it (both for the closing `if` and `<%= attrs -%>`).

I saw that ternary used here https://github.com/emberjs/data/blob/master/blueprints/model-test/qunit-files/tests/unit/__path__/__test__.js#L5 so went that route. But I agree that it isn't that readable.